src/audio/base_fw.c

```diff
@@ -391,6 +392,38 @@ static int basefw_power_state_info_get(uint32_t *data_offset, char *data)
 	return 0;
 }
 
+static int fw_config_set_force_l1_exit(const struct sof_tlv *tlv)
+{
+	const uint32_t force = tlv->value[0];
+
+	if (force) {
+		tr_info(&basefw_comp_tr, "FW config set force dmi l0 state");
+		intel_adsp_force_dmi_l0_state();
```

Comment:
You need to change the order of the two git commits. I.e. the west update needs to be first, and this commit needs to be second. Otherwise this breaks git-bisect. I.e. if somebody checks out the tree between your commits (which can happen in git-bisect), the tree cannot be build as intel_adsp_force_dmi_l0_state() is not defined).